app/src/test/java/LauncherViewModelTest.kt

```diff
@@ -49,50 +49,31 @@ class LauncherViewModelTest {
             assertTrue( "redirected to dashboard since user is already logged in", true)
         }
         launcherViewModel.redirectIfNeeded()
+        verify(preferenceRepository).shouldShowIntro
         verify(preferenceRepository).isUserLoggedIn
     }
 
-
     @Test
-    fun `should start provider screen if user has just created account`() {
+    fun `should start intro screens if user not seen intro screens`() {
```

Comment:
can we write "should show intro screens if user start application for the first time"?